.travis.yml

```diff
@@ -37,12 +53,25 @@ env:
   - TEST_DIR=core
   - TEST_DIR=util
   - TEST_DIR=config
-script: cd $TEST_DIR && cargo test --release
+  - TEST_DIR=none
+
+script:
+  - if [[ "$TEST_DIR" == "none" ]]; then cargo build --release;                fi
```

Comment:
I have hit issues with c shared libraries when trying to build a docker image based on debian that bind-mounts a grin binary I built on fedora so I am afraid if this is not static linked, it may cause issues in some linux systems.

Reply:
Do you know which shared libraries are missing? We can update the `How to use` doc (and all related doc) for the dependent shared libraries, and update the docker file.

I'm not a fans of static linking because it make the binary much bigger, and the most important:
- the bugs (especially the security related bugs) in the shared libraries need to be managed by OS security upgrading process, not managed by Grin binary.

Reply:
There wasn't a library missing, I think the issue was that the location of shared libraries differs between different linux systems. I want to try and reproduce by fiddling with the docker build but haven't had time to do so far. I don't know that the size of a binary is an issue but I can buy the argument of not having to care about OS security updates.

Reply:
@kargakis  OK.  If need help, please submit an issue to describe more detail about the problem, then I could look into it with a docker.